azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -16,14 +16,34 @@
 logger = logging.getLogger(__name__)
 
 __all__ = [
+    'RangeSchedule',
     'Solver',
     'ParallelTempering',
     'SimulatedAnnealing',
     'HardwarePlatform',
     'Tabu',
-    'QuantumMonteCarlo'
+    'QuantumMonteCarlo',
+    'PopulationAnnealing',
+    'SubstochasticMonteCarlo'
 ]
 
+class RangeSchedule:
+    def __init__(self, type:str, initial:float, final:float):
```

Comment:
Would it be possible to use a different variable name here instead of "type", since it's a protected variable name? something like `range_type` perhaps.
Also, PEP8 guidelines suggest to add spaces after the semicolon in function signatures: https://www.python.org/dev/peps/pep-0008/#variable-annotations

Reply:
"type" is by design which means the type of class obj, and this is within the class, so shall not be any confusion.
and I could not find the "add spaces after the semicolon in function signatures" in https://www.python.org/dev/peps/pep-0008/#variable-annotations
and the existing code does not have this style too. since there is a newline after semicolon of function, it make no sense to add a space